aries_cloudagent/protocols/present_proof/dif/pres_exch_handler.py

```diff
@@ -1261,9 +1261,7 @@ async def create_vp(
                 )
                 if not issuer_id and len(filtered_creds_list) == 0:
                     vp = await create_presentation(credentials=applicable_creds_list)
```

Comment:
I'm a fan of using descriptive variable names for readability. Especially in handlers. This is already shortened and pretty easy to figure out, so it's ok.